CHANGELOG.md

```diff
@@ -6,9 +6,10 @@
 * None
 
 ### Fixed
-* Allow null arguments on query. ([#872](https://github.com/realm/realm-dart/pull/872)). Fixes [#871](https://github.com/realm/realm-dart/issues/871)
-* Previously removeAt did not truncate length. ([#884](https://github.com/realm/realm-dart/pull/884)). Fixes [#883](https://github.com/realm/realm-dart/issues/883)
+* Allow null arguments on query. ([#872](https://github.com/realm/realm-dart/pull/872)). Fixes issue [#871](https://github.com/realm/realm-dart/issues/871)
+* Previously removeAt did not truncate length. ([#884](https://github.com/realm/realm-dart/pull/884)). Fixes issue [#883](https://github.com/realm/realm-dart/issues/883)
 * List.length= now throws, if you try to increase length, ([#894](https://github.com/realm/realm-dart/pull/894)).
+* Queries on lists was broken. ([#906](https://github.com/realm/realm-dart/pull/906)). Fixes issue [#909](https://github.com/realm/realm-dart/issues/909)
```

Comment:
```suggestion
* Queries on lists were broken. ([#906](https://github.com/realm/realm-dart/pull/906)). Fixes issue [#909](https://github.com/realm/realm-dart/issues/909)
```

Reply:
Also, we should probably avoid listing both the PR and the issue in the changelog entry. Since you can always get from the issue to the PR, my suggestion would be to list just the issue.